apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktOppgaveService.kt

```diff
@@ -55,16 +65,102 @@ class AktivitetspliktOppgaveService(
 ) {
     private val logger: Logger = LoggerFactory.getLogger(this.javaClass.name)
 
-    private fun hentSistEndretAktivitetspliktSomErRelevantForBrev(
-        brevdata: AktivitetspliktInformasjonBrevdata?,
-        vurderinger: AktivitetspliktVurdering?,
-    ): Tidspunkt? =
-        listOfNotNull(
-            brevdata?.kilde?.tidspunkt,
-            vurderinger?.aktivitet?.maxOfOrNull {
-                it.endret.tidspunkt
-            },
-        ).maxOrNull()
+    fun hentOppfoelgingsoppgaver(sakId: SakId): List<OppfoelgingsOppgave> {
+        val oppgaverForSak =
+            oppgaveService.hentOppgaverForSakAvType(
+                sakId,
+                listOf(OppgaveType.AKTIVITETSPLIKT, OppgaveType.AKTIVITETSPLIKT_12MND),
+            )
+
+        return oppgaverForSak.map { OppfoelgingsOppgave(!it.erUnderBehandling(), it.erFerdigstilt(), it.type) }
+    }
+
+    fun opprettOppfoelgingsoppgave(request: OpprettOppfoelgingsoppgave): UUID {
```

Comment:
Kanskje på tide å splitte opp på requesttype for vurderingtype her...

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6995/commits/443f71ce488af7e8b63b5b02efe3d99b5ccbc737